Documentation/block.md

```diff
@@ -99,3 +99,48 @@ kubectl delete -f mysql.yaml
 kubectl delete -n rook-ceph pool replicapool
 kubectl delete storageclass rook-ceph-block
 ```
+
+## Advanced Example: Erasure Coded Block Storage
```

Comment:
should this example actually be in ceph-pool-crd.md?

Reply:
No, because it is an example for block storage usage and is also containing a StorageClass.